### ISSUE TITLE
error on load

### ISSUE THREAD

**Issue by atomantic** [rghuckins/robinhood-google-sheets#2]
```Error
Cannot find method put(string,(class),(class)). (line 78).```

Perhaps this is failing because of MFA... investigating...

**Comment by westonplatter** [comment]
@atomantic could you confirm that MFA caused the error?

**Comment by atomantic** [comment]
Yes--after trying it a few more times, I realized Robinhood was sending me a lot of MFA codes :)

It appears they have interfaces for working with the MFA endpoint so it can probably be added.

**Comment by westonplatter** [comment]
Hmm, I continued to get errors. Not sure why. I removed MFA but the issue persisted.

**Comment by jWhytis01** [comment]
All,

I am also receiving the message:
_Cannot find method put(string,(class),(class)). (line 80)._

I added two lines of logging to getClassicToken_:

function getClassicToken_() {
  var url = robinhoodApiBaseUrl + '/api-token-auth/';
  var payload = {
    'username': robinhoodUsername,
    'password': robinhoodPassword
  };
  var options = {
    'method': 'post',
    'payload': payload,
    'muteHttpExceptions': true
  };
  var response = UrlFetchApp.fetch(url, options);
  var responseJson = JSON.parse(response.getContentText());
  var classicToken = responseJson.token;
  **Logger.log(responseJson)
  Logger.log(classicToken)**
  return classicToken;
}

And the log reads:
[18-08-09 11:14:02:625 EDT] {non_field_errors=[Unable to log in with provided credentials.]}
[18-08-09 11:14:02:625 EDT] undefined

Can anyone confirm their creds are working with the existing code?

Thanks,
Jason
